collector/kernel/dns/ares_expand_name.c

```diff
@@ -29,7 +29,7 @@
 ** unsigned size_t to signed long
 */
 #define CARES_MASK_SLONG 0x7FFFFFFFL
-long aresx_uztosl(size_t uznum)
```

Comment:
TODO: why do we have some of c-ares checked in under kernel/dns?  and a (small) modified version at that? In any event, grpc brings in ares which conflicts with this.